web/src/pages/speedtest.rs

```diff
@@ -245,60 +207,62 @@ impl Component for Speedtest {
 
             <p> { "
                 One of the most interesting features of AT2 is that it allows
-                for many more transactions per second (TPS) than the most
+                for many more transactions per second (TPS) than the two most
                 popular blockchains, such as Bitcoin or Ethereum.
-                Here you can actually test it, flooding the servers with
+                Indeed, it scales in " }
```

Comment:
And alos: third person singular most often takes an `s`: `linearly reduceS`, `doubling the nodes divideS`.